test/system/auto_retire_test.rb

```diff
@@ -16,7 +16,9 @@ class AutoRetireTest < ApplicationSystemTestCase
     user = users(:kyuukai)
     travel_to Time.zone.local(2020, 7, 2, 0, 0, 0) do
       VCR.use_cassette 'subscription/update' do
-        visit_with_auth scheduler_daily_auto_retire_path, 'komagata'
+        mock_env({ 'TOKEN' => 'token' }) do
+          visit_with_query scheduler_daily_auto_retire_path, { token: 'token' }
```

Comment:
```suggestion
          visit_with_query scheduler_daily_auto_retire_path, token: 'token'
```

でよさそう

そもそも、

```suggestion
          visit scheduler_daily_auto_retire_path(token: 'token')
```

でいいのでは？

Reply:
なるほど、それで書けるのですね。修正し、QueryHelperを削除しました。
[1d66013](https://github.com/fjordllc/bootcamp/pull/6970/commits/1d660135703c02b94e7caa36a4b6c99360924c81)